src/clusterfuzz/_internal/tests/appengine/handlers/upload_testcase_test.py

```diff
@@ -487,3 +487,29 @@ def test_external_duplicate(self):
         'timestamp': datetime.datetime(2021, 1, 1, 0, 0),
         'uploader_email': 'uploader@email'
     }, metadata._to_dict())
+
+  def test_trusted_uploader_post(self):
+    """Test trusted uploader post."""
+    data_types.ExternalUserPermission(
```

Comment:
You may need to add the email to https://github.com/google/clusterfuzz/blob/7f94ec184870762c04046b23c24ff1a0424133b4/src/clusterfuzz/_internal/datastore/data_types.py#L787 as well. 

Look in other tests for examples of setting up fake Config objects.